views/404.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>404</title>
```

Comment:
"404" não é um título que usuários leigos entendem em geral, sugiro dar um título como "Page not found"